content/en/docs/Installation/pipelines.md

```diff
@@ -0,0 +1,17 @@
+<!--
+---
+title: "Install Tekton Pipelines"
+linkTitle: "Tekton Pipelines"
+weight: 2
+description: >
+  Install Tekton Pipelines on your cluster
+---
+-->
+
+{{% readfile "/docs/Pipelines/install.md" %}}
```

Comment:
Ah, nice, I didn't know you could do this.
I think we should mention somewhere that this is the install guide for Tekton on the `main` branch.
Version-specific guides are available on GitHub.

Reply:
Ah, cool, I didn't realise this, nice work!